src/ansys/mapdl/core/_commands/aux2_/bin_manip.py

```diff
@@ -21,323 +21,315 @@
-        .FULL file generated by a harmonic analysis.
-
-        When copying a .FULL file, the rows and columns corresponding to
-        specified constraints (e.g., D commands) are eliminated from the system
-        of equations and therefore are not written to the .PS file. In
-        addition, rows and columns corresponding to eliminated (slave) degrees
-        of freedom from coupling and/or constraint equations (e.g., CE, CP
-        commands) are eliminated from the system of equations and are not
-        written to the .PS file. The DOFs that are eliminated from any coupling
-        and/or constraint equations are determined internally by the solution
-        code and may not match what you specified via the CE/CP (or similar)
-        commands.
-
-        When copying a .FULL file, only the upper triangular part of the matrix
-        will be written to the .PS  file if the matrix is symmetric; the full
-        matrix is written if the matrix is unsymmetric. Only matrix
-        coefficients that are greater than zero will be written.
-
-        The WRFULL command, in conjunction with the SOLVE command, can be used
-        to generate the assembled global matrix file and eliminate the equation
-        solution process and results output process.
+        This command is used to copy a matrix from the assembled global matrix file ( :file:`.FULL` file) as
+        specified on the :ref:`file` command and write it in a postscript format to a new file named
+        ``Fname``. ``Ext`` (defaults to :file:`Jobname.PS` ). The matrix is symbolized by a grid in which
+        the black or colored squares represent the nonzero coefficients of the matrix. The :file:`.FULL`
+        file must be available for this command to work properly.
+
+        If the matrix is large, it may be difficult to display the postscript file. In this case, use
+        ``Color`` = BLACK to reduce the postscript file size.
+
+        The assembled global matrix file is created during solution depending on the analysis type, equation
+        solver, and other solution options. By default, the assembled global matrix file is never deleted at
+        the end of solution. For most analysis types, the Sparse direct solver and the ICCG solver write a
+        :file:`.FULL` file. All mode extraction methods used for buckling and modal analyses write a
+        properly formatted :file:`.FULL` file to be used with the :ref:`psmat` command.
+
+        When copying the stiffness matrix for transient and harmonic analyses, be aware that the element
+        mass matrix values (and possibly element damping matrix values) are incorporated into the globally
+        assembled stiffness matrix. Thus, the globally assembled stiffness matrix represents more than the
+        stiffness of the model for these analysis types. Please refer to the `Theory Reference
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_thry/thy_biblio.html>`_  Mechanical APDL
```

Comment:
Shouldn't this be pointing to v251?

Reply:
Or are we using v232 as base?

Reply:
yeah... I think we are using v232 as base. It is fine for the moment. That gives us the chance to later update and see how much work it is.

Reply:
I confirm, we are using v23.2 as base